x/data/features/msg_attest.feature

```diff
@@ -0,0 +1,21 @@
+Feature: MsgAttest
+
+  Scenario Outline: attestor must be valid
+    Given an attestor of "<attestor>"
+    And a valid content hash
+    When the message is validated
+    Then an error of "<error>"
+
+    Examples:
+    | attestor                                      | error                                                                   |
```

Comment:
one way we could do this to cover all cases msg validation would be something like this
```suggestion
    | message                                      | error                                                                   |
    | {"attestor":"foo"}.                      | decoding bech32 failed                                |
```

where message is proto json

Reply:
Any attempt to use json as the value produces the following error:

```
internal error: no matches found when matching ^a\s+content\s+hash\s+of\s+"([^"]*)"$
```

https://github.com/regen-network/gocuke/blob/main/run_step.go#L39-L42

Reply:
Ah... well you just have to manually register the step with a regex then